lesson_20/closureFunction.js

```diff
@@ -0,0 +1,18 @@
+// function structureUserInfo(userName) {
+//   return function (userRole) {
+//     return {
+//       role: userRole,
+//       name: userName
+//     };
+//   };
+// }
+
+const structureUserInfo = (userName) => (
+  (userRole) => (
+    {
+      role: userRole,
+      name: userName
+    })
```

Comment:
Pretty nice! We could simplify it `return role => ({ role, name })`

Reply:
done